src/test/java/uk/gov/hmcts/reform/wacaseeventhandler/entity/CaseEventMessageEntityTest.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmcts.reform.wacaseeventhandler.entity;
+
+import java.time.LocalDateTime;
+import java.util.Map;
+
+public class CaseEventMessageEntityTest {
```

Comment:
I named it `CaseEventMessageEntityTest` since I think it would be confusing for anyone else trying to find the class. I created it in the test folder not the main one. But if you insist I could rename it to `CaseEventMessageEntityCreatorTest` but I would have to rename the main file to that as well to avoid confusion

Reply:
ok done